tutorials/nwbc-cflp-integration/nwbc-cflp-integration.md

```diff
@@ -0,0 +1,131 @@
+---
+parser: v2
+auto_validation: true
+time: 20
+tags: [ tutorial>beginner, software-product-function>ui-sap-business-client--nwbc-]
+primary_tag: software-product-function>ui-sap-business-client--nwbc-
+author_name: Sandra Thimme
+author_profile: https://github.com/sandra36
+---
+
+# Launch native SAP GUI for Windows Applications in the Cloud with SAP Business Client
```

Comment:
Please use title case (->native)

Reply:
This comment has not been addressed.  Please change "native" to "Native" as it should be title case.